pass-client-api/src/main/java/org/dataconservancy/pass/client/PassClient.java

```diff
@@ -89,11 +81,14 @@
     
     
     /**
-     * Retrieves URIs for ALL MATCHING RECORDS by matching the entity type and filtering by the field
+     * Retrieves URIs for MULTIPLE MATCHING RECORDS by matching the entity type and filtering by the field
      * specified using the value provided. For example, to find Deposits using a Repository.id:
      * 
      *    URI repositoryId = new URI("https://example.com/fedora/repositories/3");
      *    Set<URI> entityUris = findByAttribute(Deposit.class, "repository", repositoryId);
+     *    
+     * By default this will return a maximum of 3000 matching records, unless the pass.elasticsearch.limit
```

Comment:
@birkland mind if I log an issue for this? I want to wrap this up so people can use it and this will add time to do tests and what not. Or do you see this is vital?

Reply:
Sure, another is fine.  I think the only consumer that might possibly use large responses might be the COEUS loader (@jrmartino ?), but even then I'm not sure.  So a separate issue is fine.